.storybook/preview.js

```diff
@@ -39,6 +39,7 @@ const SORT_ORDER = {
   Layout: {},
   Forms: {},
   Components: {},
+  Patterns: {},
```

Comment:
I'm thinking of adding the component into this new Storybook section to match Figma ([Circuit UI Patterns page](https://www.figma.com/file/2yHC0dK9pFEhLsMxk62XZY/?node-id=3120%3A2322))

Reply:
I've wanted to re-arrange and rename those sections for a while but could never come up with a better organization. I like the idea of aligning them with Figma 👍🏻 Perhaps we can re-organize this in a separate PR?

Reply:
So you mean, keep it in `Components` in this PR and move them to `Patterns` later? Because it's already there, and I don't think we have other components in that category yet (do we?)

Reply:
You can also move it to `Patterns` right away. I didn't check how the organisation looks like in Figma 🙈